test/twofishes/client_test.rb

```diff
@@ -3,35 +3,81 @@
 module Twofishes
   class Zurich
     def self.geocode(location)
-      YAML::load_file('test/fixtures/response.yaml')
+      if location.responseIncludes == [ResponseIncludes::PARENTS]
```

Comment:
I agree. But I fear we're testing the fake geocode method instead of the real one.